Mathlib/Topology/Category/Stonean/EffectiveEpi.lean

```diff
@@ -134,13 +127,4 @@ theorem effectiveEpiFamily_of_jointly_surjective
     EffectiveEpiFamily X π :=
   ((effectiveEpiFamily_tfae X π).out 2 0).mp surj
 
-open CompHaus Functor
-
-theorem _root_.CategoryTheory.EffectiveEpiFamily.toCompHaus
-    {α : Type} [Finite α] {B : Stonean.{u}}
```

Comment:
This theorem can now be proved by `infer_instance`. It was also apparently never used.